.adops/package-update.yml

```diff
@@ -0,0 +1,89 @@
+trigger: none
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+  demands: project_navy
+
+resources:
+  pipelines:
+    - pipeline: packageRelease
+      source: 'Package release'
+      trigger:
+        branches:
+          include:
+            - master
+        stages:
+          - Prerelease
+
+variables:
+  # This is Yarn 1, it has different cache behaviour than Yarn 2/3!!!
```

Comment:
maybe I missed something - but isn't this repo using yarn v3?

Reply:
Updated with the same cache path as that one used in the build pipeline.